webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -178,16 +217,42 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
         {canModifyBindings && Loadable.getOrElse([], resourceManagers).length > 0 && (
           <>
             <Divider />
-            Namespace Bindings:
+            Namespace Bindings
             <Body inactive>
               Note: If you leave the Namespace name blank, the workspace will be bound to the
               default Namespace configured in the Master Config.
             </Body>
             <>
               {Loadable.getOrElse([], resourceManagers).map((name) => (
-                <Form.Item key={name} label={name} name={['bindings', name]}>
-                  <Input maxLength={63} />
-                </Form.Item>
+                <Fragment key={name}>
+                  <Form.Item label={name} name={['bindings', name, 'namespace']}>
+                    <Input
+                      disabled={watchBindings?.[name]?.['autoCreateNamespace'] ?? false}
+                      maxLength={63}
+                    />
+                  </Form.Item>
+                  {info.branding === BrandingType.HPE && (
+                    <>
+                      <Form.Item
+                        label="Auto Create Namespace"
+                        name={['bindings', name, 'autoCreateNamespace']}
+                        valuePropName="checked">
+                        <Toggle
+                          onChange={() => form.setFieldValue(['resourceQuotas', name], undefined)}
+                        />
+                      </Form.Item>
+                      {(workspaceId !== undefined ||
```

Comment:
`Resource Quota` field is visible even though `Auto Create Namespace` toggle is off. can we hide it when its off?

Reply:
Is it visible for you if autocreate toggle is off while creating the workspace? While editing it, it is expected behavior to show the resource quota even if auto create toggle is off

Reply:
yeah its visible. if its expected, i think its ok